src/main/java/com/demo/crud/operation/service/DataServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package com.demo.crud.operation.service;
```

Comment:
Try implementing Exception handling and logging in this next.
With this we can return different status code if a exception is catch 
and with logging we can easily debug the code.

Reply:
Yes logging is left will implement next